src/git/remotes/github.ts

```diff
@@ -20,7 +20,7 @@ const rangeRegex = /^L(\d+)(?:-L(\d+))?$/;
 const authProvider = Object.freeze({ id: 'github', scopes: ['repo', 'read:user', 'user:email'] });
 
 export class GitHubRemote extends RichRemoteProvider {
-	protected get authProvider() {
+	protected get authProvider(): { id: string; scopes: string[] } {
```

Comment:
Need return type because type inference causes the following compile error.
```
  ERROR in src/git/remotes/github-enterprise.ts:6:25
  TS2416: Property 'authProvider' in type 'GithubEnterpriseRemote' is not assignable to the same property in base type 'GitHubRemote'.
    Type 'Readonly<{ id: "github-enterprise"; scopes: string[]; }>' is not assignable to type 'Readonly<{ id: "github"; scopes: string[]; }>'.
      Types of property 'id' are incompatible.
        Type '"github-enterprise"' is not assignable to type '"github"'.
```